server/util/monitoring/monitoring.go

```diff
@@ -45,14 +49,20 @@ func RegisterMonitoringHandlers(mux *http.ServeMux) {
 
 	// Flagz page
 	handle("/flagz", http.HandlerFunc(flagz.ServeHTTP))
+
+	// Channelz page
+	handle("/", channelz.CreateHandler("/", fmt.Sprintf("%s:%d", env.GetListenAddr(), grpc_server.Port())))
```

Comment:
I think /channelz should show up on the monitoring port, similar to /statusz and /healthz?